hms-app/src/components/card-details/IdeaDetails.tsx

```diff
@@ -217,42 +217,26 @@ export default function IdeaDetails(props: IProps) {
                 </Group>
               </Card.Section>
 
-              {type === IdeaCardType.IdeaPortal && (
-                <Accordion
-                  state={accordionState}
-                  onChange={setAccordionState.setState}
+              <Accordion
+                state={accordionState}
+                onChange={setAccordionState.setState}
+              >
+                <Accordion.Item
+                  className={classes.noBorderAccordion}
+                  label={!accordionState['0'] ? 'Show details' : 'Hide details'}
                 >
-                  <Accordion.Item
-                    className={classes.noBorderAccordion}
-                    label={
-                      !accordionState['0'] ? 'Show details' : 'Hide details'
-                    }
-                  >
-                    <div>{ideaDetails()}</div>
+                  <div>{ideaDetails()}</div>
 
+                  {type === IdeaCardType.IdeaPortal && (
                     <Group mt="xs" position={'right'} style={{ paddingTop: 5 }}>
                       <Button variant={'outline'} color={'yellow'}>
                         Add to Favorites
                       </Button>
                       <Button>Participate</Button>
```

Comment:
should this only say participate if it is not you idea?

Reply:
in fact couldn't the design depend purely only who you are in relation to the Idea?  If you are an admin or the owner of the Idea then it should show the Edit/Delete buttons and if you are not then it should show Participate?

And the participate button should show something different depending on if you are already participating in the Idea right?

Reply:
Should the Participate button always be there even?

Reply:
And lastly I'm always more of a fan of using icons/glyphs instead of text for buttons.  It allows them to be consistently sized for all languages and means you don't need to bother translating

Reply:
I see your point and it somehow makes sense 

not 100% sure if the "role" dependencie would be that much better

Reply:
well it would mean that only the area of the buttons would need to be dynamic.  It will also ensure that whenever/wherever I look at an Idea description it will always appear the same to me